svelte-case-study.md

```diff
@@ -0,0 +1,203 @@
+
+Svelte also performs well when creating Multi Page Applications (MPAs). When building an MPA in Svelte, you would typically use a routing library such as Sapper to handle navigation between pages, with each page being built as a separate Svelte component. The components are loaded and unloaded based on the current URL, with data management handled separately for each page.
+
+## Weaknesses
+
+Svelte's weaknesses mainly stem from it being a relatively new framework compared to its largest competitors. It has a smaller community of developers and a more limited number of available resources. It is also not as widely used as other popular JavaScript frameworks, meaning that it may be more difficult to find experienced developers, tools, and resources to work with compared to competing frameworks. It also does not have an abundance of available packages, plugins and tools, which may prevent you from performing specific tasks or adding specific functionality to your apps.
+
+As mentioned previously, Svelte uses a relatively simple syntax. It does, however, also have a very different approach to building web applications compared to other popular frameworks, mainly by not utilizing a virtual DOM. This can result in a steep learning curve, especially for experienced developers who are used to the virtual DOM approach.
+
+In conclusion, while Svelte has some advantages over other frameworks, it also has its own set of limitations. Whether or not Svelte is the right choice for your project will depend on your specific requirements, the size and complexity of the project, and the experience and skill level of your development team. It may be best to consider using Svelte for smaller projects or as a part of a larger, more complex application built with other frameworks.
+
+## Comparison
+
+Although Svelte, React, Vue.js and Angular are all popular JavaScript frameworks for building web applications, they have some very different design philosophies, strengths and weaknesses depending on their use.
+
+Svelte has a relatively simple syntax and a different approach to building web applications, making it relatively easy to learn. React has a steeper learning curve due to its use of a virtual DOM, although it has a large and supportive community. Vue.js has a similar learning curve to React, but with a simpler syntax. Angular has the steepest learning curve of the four frameworks, due to its complex syntax and larger feature set. They also all differ in scope, as Svelte is mainly meant for smaller applications, where as React and Vue are tailored towards medium to large-size applications which do not require many different pages. Angular on the other hand is tailored towards very large applications with many different views, pages and URLs.
```

Comment:
is it **whereas** or where as? i am not 100% sure  but check also if medium**-** to large-size or simply medium to large-size****

Reply:
![Screenshot_20230214-150918](https://user-images.githubusercontent.com/30121693/218762628-fec3911c-de85-4791-82b8-93417e17dc83.png)
Good catch, thanks